### ISSUE TITLE
Config does not have enough customization options

### ISSUE THREAD

**Issue by JackStanley** [mastercoms/tf2cfg#7]
1. Comms right after cvars:
`m_mousespeed 0     // disable Windows mouse acceleration`

^ I can definitely see the cvar and comm

```
// --- Input ---
// determine key time from a smoothed frametime
// this does not delay your keyboard input
in_usekeyboardsampletime 1
// -- Mouse --
// disable Windows mouse acceleration
//m_mousespeed 0
```
^ It's hard to see through code like this:

2.Also the config just seems to be more personal than for public, some things (like Threading; shadows) should have a better customization (options for 2 core cpu's for e.g), and some things should be set up to have a lil game advantage (for e.g dsp_water ought to be set to 0; uncomment mat_phong 0 cvar and pyrovision effects)
Comanglia's config just seems to have more disabled cvars

3.I though mat_picmip has a 2 as minimum, not 4 (I can't be sure, but 4 works with sv_cheats 1 only and breaks mat_filertextures)

4.Some things are just not about fps at all and they are to be in specific file, still. things like cl_mvm_wave_status_visible_during_wave 1 should be in main file as they give a bit of game advantage

tl;dr : This config is just hard to see through and some parts need to be reconsidered

**Comment by mastercoms** [comment]
I've been working on a rewrite of the config to make it a lot clearer

**Comment by mastercoms** [comment]
Fixed mat_picmip being 4, that was a typo. 
cl_mvm_wave_status_visible_during_wave 1 is already in the autoexec.cfg, enabled by default.

The rewrite of the config will include a comment structure rewrite and look over defaults/possible values for different types of systems.

**Comment by fagoatse** [comment]
I think comanglia's structure is very good. Everything is separated from each other. If you don't want to go that far please just group all cvars that are a matter of preference such as sens, viewmodel fov, etc and separate them from the others.

**Comment by mastercoms** [comment]
I've been investigating further and I have come to the conclusion that the multicore options will benefit even 2 core CPUs. But I'd like someone to verify this.

**Comment by JackStanley** [comment]
Are they stable for 2 core cpus?

**Comment by mastercoms** [comment]
Here are the ones I feel are beneficial (and stable) to use on 2 core CPUs:

cl_threaded_client_leaf_system 1
r_threaded_renderables 1
r_threaded_particles 1
mat_queue_mode 2
r_queued_post_processing 1 (if your GPU is fast enough to work with it)

Disable these on 2 core:
r_threaded_client_shadow_manager 0
r_queued_decals 0 (disabled anyway for even 4 cores due to stability issues)
r_queued_ropes 0

**Comment by mastercoms** [comment]
See https://github.com/mastercoms/tf2cfg/tree/4.0.0
